lenovo/thinkpad/x1-gen6/default.nix

```diff
@@ -0,0 +1,41 @@
+# pulled out all the x1 specific things
+# for nixos-hardware
+
+{ config, pkgs, ... }:
+
+{
+
+  boot.kernelModules = [ "acpi" "thinkpad-acpi" "acpi-call" ];
+  boot.extraModulePackages = [ config.boot.kernelPackages.acpi_call ];
+  boot.extraModprobeConfig = "options thinkpad_acpi experimental=1 fan_control=1";
+
+  environment.systemPackages = with pkgs; [
```

Comment:
If you import the default module this is implied by that (in `common/pc/laptop`)